### DISCUSSION TITLE
Нeadings indicator does not export to pdf

### DISCUSSION THREAD

**Discussion by squarepusher** [Akifyss/obsidian-border#311] [Q&A] [answered]
Hello. First of all, I want to thank you for the great theme you have created. I have some difficulty with exporting notes. When exporting to PDF, the header indicators and other theme colours (bold, etc) are not saved. I understand this is not normal. And some users here even asked to disable it. But I like this markup and colour scheme. And I would also like to have it all on the pdf file page when exporting. I am an inexperienced Obsidian user and don't know how to fix this. Maybe it is related to some CSS added before I installed this theme. I would appreciate any help you can give me. Thank you, and have a nice day.  
![Untitled-3](https://github.com/user-attachments/assets/66f96b82-19fa-4961-869d-460822d9a8e2)

**Comment by m-zaya** [comment]
> Hello. First of all, I want to thank you for the great theme you have created. I have some difficulty with exporting notes. When exporting to PDF, the header indicators and other theme colours (bold, etc) are not saved. I understand this is not normal. And some users here even asked to disable it. But I like this markup and colour scheme. And I would also like to have it all on the pdf file page when exporting. I am an inexperienced Obsidian user and don't know how to fix this. Maybe it is related to some CSS added before I installed this theme. I would appreciate any help you can give me. Thank you, and have a nice day. ![Untitled-3](https://private-user-images.githubusercontent.com/6873593/349043999-66f96b82-19fa-4961-869d-460822d9a8e2.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0NTcxODQsIm5iZiI6MTcyMTQ1Njg4NCwicGF0aCI6Ii82ODczNTkzLzM0OTA0Mzk5OS02NmY5NmI4Mi0xOWZhLTQ5NjEtODY5ZC00NjA4MjJkOWE4ZTIuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MjBUMDYyODA0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTRlYzhmMDQxOGM0MWExYzc2M2ZhODkxYzUzYjBjMTQwYTI2MjJhMjE1MzNmZDY3YzBiYmI1NDM4N2ZmY2RjMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.0R7lREarSpUas2k2izo2VORBi4vp2xXDKIPJs5_Xey0)

I'm having the same issue!

I love the little heading indicators, and when I tried exporting to PDF, they were gone.

The only one that appears is if I enable the 'Include file name as title' option, and that header has the little color marker.

Otherwise, all the other headings' color markers don't export to PDF.

I absolutely love this theme, and I hope this gets fixed soon.

**Comment by Akifyss** [comment, marked as answer]
The new version has fixed this issue. You can try updating it.

**Reply by squarepusher** [reply]
Yeah, now it's really working. Thank you from the bottom of my heart. All the best to you!!!
